explorer/Explorer.scss

```diff
@@ -30,10 +30,10 @@ html.IsInIframe #ExplorerContainer {
     justify-content: space-around;
 
     .ExplorerTitle {
-        font-size: 28px;
+        font-size: max(16px, min(1.5vw, 19px));
+        line-height: max(18px, min(1.7vw, 23px));
```

Comment:
Quick tip: You can also use the [`clamp()` CSS function](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp()) for such a term.

> `clamp(MIN, VAL, MAX)` is resolved as `max()(MIN, min()(VAL, MAX))`

It also has [the same browser support](https://caniuse.com/css-math-functions) as `min` and `max`.

Reply:
Way more readable! I'll change it.